Lombiq.Hosting.Azure.ApplicationInsights/Extensions/ApplicationInsightsInitializerExtensions.cs

```diff
@@ -16,39 +17,46 @@ public static class ApplicationInsightsInitializerExtensions
     /// <summary>
     /// Initializes Application Insights for Orchard Core. Should be used in the application Program.cs file.
     /// </summary>
-    public static IServiceCollection AddOrchardCoreApplicationInsightsTelemetry(
-        this IServiceCollection services,
+    public static OrchardCoreBuilder AddOrchardCoreApplicationInsightsTelemetry(
+        this OrchardCoreBuilder builder,
+        IServiceCollection services,
```

Comment:
This is always `build.Services`, right? Because then the parameter is not needed, you can take if from `builder`.